techniques/html/H99.html

```diff
@@ -153,13 +153,13 @@ <h3>Example 3: Page list in EPUB 2 and DAISY publications</h3>
 &lt;/ncx&gt;</code></pre>
 			</section>
 			<section class="example" id="example-4-go-to-page-mechanism">
-				<h3>Example 4: Go to page mechanism</h3>
-				<p>A mechanism provides the ability to enter a page number as an input and reposition the user to the matching target page location.</p>
+				<h3>事例 4: Go to ページメカニズム</h3>
+				<p>メカニズムにより、ページ番号を入力として入力し、一致するターゲットページの場所に利用者を再配置する機能が提供される。</p>
```

Comment:
「ページ番号を入力として入力し」という表現が気になりました。意訳になりますが、「ページ番号を入力し」でよいかもしれません。また、「(利用者を) 再配置する」という表現も気になりました。

例えば以下のようにするのはいかがでしょうか？
```suggestion
				<p>メカニズムにより、ページ番号を入力し、一致するターゲットページの場所に利用者を移動させる機能が提供される。</p>
```

Reply:
enterとinputをわざわざ使っているのはきっと意味がある、かもしれません。
また、repositionは受け身にしなくてもいいかな…と。
こうだとどうでしょう？

```suggestion
				<p>メカニズムにより、入力としてページ番号を記入し、一致するターゲットページの場所に利用者を移動する機能が提供される。</p>
```